app/src/main/java/org/simple/clinic/security/pin/PinEntryCardView.kt

```diff
@@ -120,10 +145,11 @@ class PinEntryCardView(context: Context, attrs: AttributeSet) : CardView(context
   override fun onDetachedFromWindow() {
     pinEntryLockedCountdown?.cancel()
     delegate.stop()
+    binding = null
     super.onDetachedFromWindow()
   }
 
-  override fun onSaveInstanceState(): Parcelable? {
+  override fun onSaveInstanceState(): Parcelable {
```

Comment:
Because it's no longer necessary. With every AppCompat update, they are adding more `Nullable` & `NonNull` annotations. So now it's no longer a nullable property.